docs/docs/configuration-parameters.md

```diff
@@ -71,3 +71,5 @@ It is sufficient to specify just one of the `zk` or `masters` field. If both are
 `SOAMinttl` is the minimum TTL field in the SOA record for the Mesos domain. For details, see the [RFC-2308](https://tools.ietf.org/html/rfc2308). The default value is `60`.
 
 `recurseon` controls if the DNS replies for names in the Mesos domain will indicate that recursion is available. The default value is `true`. 
+
+`enforceDNS952` will enforce an older, more strict set of rules for DNS labels. For details, see the [RFC-952](https://tools.ietf.org/html/rfc952). The default value is `false`.
```

Comment:
I think this should be `enforceRFC952`. Also, don't we want the default to be true for backwards compatibility?